docs/guides/config.json_directives.md

```diff
@@ -0,0 +1,143 @@
+# Getting the bot started
+
+## config.json directives
+
+config.json, a file that can store data, that is used by Concord on runtime. You will probably wish to set it up, so, here is a guide explaining what are the purposes of everything in there.
```

Comment:
```suggestion
The `config.json` file is a configuration file that can be used to start your bots from. Since it's read at runtime, it is especially useful if you want to avoid having to recompile your bot application for each small configuration that gets added. This guide outlines some of the configuration options you can modify, and also how to add your own.
```